python/nav/ipdevpoll/shadows/__init__.py

```diff
@@ -21,6 +21,9 @@
 database I/O.
 
 """
+
+# ruff: noqa: F401
```

Comment:
Here I would like to add an explanation on why we import so much which looks "unused"

Reply:
As proven by other changes you've made, you could make things more explicit by adding all symbols that are meant to be exported from a module to an `__all__`.  This shows the intent that even imported symbols are meant for further export, even if they are not used locally.